library/core/src/slice/ascii.rs

```diff
@@ -432,3 +439,28 @@ const fn is_ascii(s: &[u8]) -> bool {
 
     !contains_nonascii(last_word)
 }
+
+#[cfg(kani)]
+#[unstable(feature = "kani", issue = "none")]
+pub mod verify {
+    use super::*;
+
+    #[kani::proof]
+    #[kani::unwind(8)]
```

Comment:
Same question on the unwind here.

Reply:
'is_ascii' calls 'is_ascii_simple' to handle base case where the loop in 'is_ascii_simple' execute at most sizeof(usize) number of times. So we unwind it 8 times.